test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 freezegun
 odoo_test_helper
 responses
+odoo14-addon-account-invoice-transmit-method @git+https://github.com/akretion/account-invoicing@14-mig-transmit_method#subdirectory=setup/account_invoice_transmit_method
```

Comment:
Is it a common practice to target a module that is not in an OCA repo ?

Reply:
Thanks @mmequignon for your comment. We are working on this.

Reply:
Thanks @mmequignon for the comment. That dependency has been merged by https://github.com/OCA/account-invoicing/pull/823. So the `test-requirements.txt` is updated just now.